web/src/api/moreCast2API.ts

```diff
@@ -187,9 +187,9 @@ export const marshalMoreCast2ForecastRecords = (forecasts: MoreCast2ForecastRow[
       for_date: forecast.forDate.toMillis(),
       precip: forecast.precip.value,
       rh: forecast.rh.value,
-      temp: forecast.temp.value,
+      temp: Math.round(forecast.temp.value),
```

Comment:
I'm not sure we need to be rounding values here. I think I would error on the side of maintaining precision in the event that a forecaster is using persistence forecasting or values from one of the weather models.

Or is the reason to do the rounding to prevent decimals from being sent to and appearing in the WFWX forecasting app?

Reply:
Yes, to prevent decimals from appearing in WFWX, and following my interpretation of the acceptance criteria "Forecaster can only forecast integers for temp and wind speed". Without rounding, the front end would show integers but they would technically be forecasting floats if they're using models.
I also figured the discrepancy between Morecast and WFWX might be puzzling. 

Open to changing it of course, that was just my interpretation of the desired functionality

Reply:
Having WF1 and Morecast match is good enough reason for me! Thanks for the clarification!